pisa/stages/osc/two_nu_osc.py

```diff
@@ -44,7 +44,7 @@ def __init__(self,
                 ):
 
         expected_params = (
-            'theta',
+            'theta23',
```

Comment:
The initialisation within the `init_test` function at the bottom of the module needs to reflect this naming change in order to get the service test to pass again.

Reply:
Fixed